### ISSUE TITLE
How to get more than one labels to input?

### ISSUE THREAD

**Issue by h612** [vlfeat/matconvnet#978]
Hello,
I want to calculate 2 losses
if one is below a Th, train the network on the other one
or reduce both losses somehow
I've tried 
```
Getbatch
inputs = {'input', images, 'label', labels,'labelPD',labelPD} ;
```
and at loss Layer
```
% c has two label per image
net.addLayer('objective', dagnn.Loss('loss','logistic'), {'classifier', 'label','labelPD'}, {'objective'}, {});
```
But I'm afraid this way I'll have to change the entire vl_nnloss- not sure how it can be done?
Kindly assist.

**Comment by h612** [comment]
I've tried the following code for two regression losses (one is for count of objects in an image 'label'- and other is density of objects 'pd')
**```
net.addLayer('fc7', dagnn.Conv('size', [1 1 400 1], 'hasBias', true, 'stride', [1, 1], 'pad', [0 0 0 0]), {'relu7'}, {'**prediction**'},  {'conv8f'  'conv8b'});
net.addLayer('error1', dagnn.Loss('loss', 'classerror'), {'prediction','**pd**'}, 'objective2') ;
net.addLayer('l2_loss', dagnn.L2Loss(), {'prediction', '**label**'}, {'objective1'});
net.addLayer('error2', dagnn.Loss('loss', 'classerror'), {'prediction','label'}, 'error') ;
```
**
but the training doesn't seem to converge. I've attached the error objective plot. Please help.
